static_content/_includes/footer.html

```diff
@@ -5,7 +5,7 @@
       <div class="col-xs-6 col-sm-4 col-lg-2">
         <h2>Apply</h2>
         <ul>
-          <li><a href="/apply/">Application form</a></li>
+          <li><a href="#">Application form</a></li>
```

Comment:
Not sure where this comes in, but it might be our missing url in the html footer

Reply:
Oops, accidentally changed this.

Reply:
Nevermind this comment: I manually update the footer/header templates from `portality/`; the footer & header in `static_content` are outdated now! But fixed and pushed :)